share/functions/__fish_whatis_current_token.fish

```diff
@@ -0,0 +1,19 @@
+# This function is typically bound to Alt-W, it is used to list man page entries
+# for the command under the cursor.
+function __fish_whatis_current_token -d "Show man page entries related to the token under the cursor"
+    set tok (commandline -pt)
+
+    if test $tok[1]
+        printf "\n"
+        whatis $tok[1]
+
+        set -l line_count (count (fish_prompt))
+        if test $line_count -gt 1
```

Comment:
This `if` is not necessary I guess. Looks like `__fish_list_current_token` is using it as well, not sure why.

Reply:
I know this has already been merged, but I do actually think this `if` is necessary:

When `fish_prompt` is a single line, this resolves to `seq 2 1` which produces
```fish
2
1
```
which will end up printing two newlines! This is definitely not what we want. When `fish_prompt` is two lines, we get `seq 2 2` which does indeed produce simply `2` and prints one extra newline, as desired.

Reply:
>When fish_prompt is a single line, this resolves to seq 2 1 which produces

Aaaah! That's an incompatibility in seq implementations! GNU `seq` produces nothing in that case, while BSD seq does indeed produce 2 1.